src/components/Radio/Radio.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Radio } from './index';
+import pkg from './package.json';
+import { buildExcludeArgTypes } from '../../storybook/utils';
+
+export default {
+  title: 'Components/Radio',
+  component: Radio,
+  parameters: {
+    pkg
+  },
+  argTypes: buildExcludeArgTypes(['className'])
+} as ComponentMeta<typeof Radio>;
+
+const Template: ComponentStory<typeof Radio> = (args) => {
+  return <Radio {...args}>
```

Comment:
It would be nice to have more than 1 radio element here, as it can't be "unchecked" right now if it's checked manually